conf/config-default.yaml

```diff
@@ -89,6 +89,8 @@ apisix:
       role: viewer
 
   delete_uri_tail_slash: false    # delete the '/' at the end of the URI
+  global_rule_skip_internal_api: true    # does not run global rule in internal apis
```

Comment:
Can we add test when global_rule_skip_internal_api is false?

Reply:
We need to add test when global_rule_skip_internal_api is false.

Reply:
> We need to add test when global_rule_skip_internal_api is false.

sure, test failed now, submit it after solving it.

Reply:
@spacewander  teat case added. please help review when you have time, thanks.